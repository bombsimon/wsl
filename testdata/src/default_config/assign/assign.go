package testpkg

func strictAppend() {
	s := []string{}
	s = append(s, "a")
	s = append(s, "b")
	x := "c"
	s = append(s, x)
	y := "e"
	s = append(s, "d") // want "missing whitespace decreases readability"
	s = append(s, y)
}

func incDec() {
	x := 1
	x++
	x--
	y := x

	_ = y
}
