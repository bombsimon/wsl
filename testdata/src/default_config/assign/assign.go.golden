package testpkg

type T struct {
	I int
}

func NewT() *T {
	return &T{}
}

func (*T) Fn() int {
	return 1
}

func strictAppend() {
	s := []string{}
	s = append(s, "a")
	s = append(s, "b")
	x := "c"
	s = append(s, x)
	y := "e"

	s = append(s, "d") // want "missing whitespace decreases readability"
	s = append(s, y)
}

func incDec() {
	x := 1
	x++
	x--
	y := x

	_ = y
}

func assignAndCall() {
	t1 := NewT()
	t2 := NewT()

	t1.Fn()
	t2.I = t1.Fn()
}
